src/compressed_tensors/compressors/quantized_compressors/base.py

```diff
@@ -234,3 +233,22 @@ def decompress_from_state_dict(
                 )
                 weight_data["weight"] = decompressed
                 yield module_path, weight_data
+
+    def decompress_module_from_state_dict(
+        self,
+        prefix: str,
+        state_dict: Dict[str, torch.Tensor],
+        scheme: QuantizationScheme,
+    ) -> Generator[Tuple[str, Dict[str, torch.Tensor]], None, None]:
```

Comment:
This function is returning a dict, right?
```suggestion
    ) -> Dict[str, torch.Tensor]:
```

Reply:
Yes